src/handlers/cell.ts

```diff
@@ -10,12 +10,15 @@ import { Doc, Editor } from 'codemirror';
 import { Breakpoints, SessionTypes } from '../breakpoints';
 
 import { IDisposable } from '@phosphor/disposable';
+
 import { Debugger } from '../debugger';
+
 import { IDebugger } from '../tokens';
 
 import { Signal } from '@phosphor/signaling';
 
 const LINE_HIGHLIGHT_CLASS = 'jp-breakpoint-line-highlight';
+const GUTTER_HOVER_CLASS = 'jp-breakpoint-hover';
```

Comment:
Typically for transient classes, we use the `mod` convention, so I would suggest calling this `jp-mod-hover`.